File: generators/java/src/main/java/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/DomainArchitectureMapping.java
package dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

import java.util.LinkedHashSet;

/**
 * Java class for the architecture-mapping
 */
@Getter
@ToString
public class DomainArchitectureMapping {

    private long version;
    private String context;
    @JsonProperty("server_info")
    private LinkedHashSet<ServerInfo> serverInfo;
    private LinkedHashSet<DomainArchitectureMappingObject> objects;
}

File: src/main/kotlin/dqualizer/dqlang/archive/loadtesttranslator/dqlang/domainarchitecturemapping/DomainArchitectureMapping.kt
package dqualizer.dqlang.archive.loadtesttranslator.dqlang.domainarchitecturemapping

/**
 * Java class for the architecture-mapping
 */
import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class DomainArchitectureMapping {
    private val version: Long = 0
    private val context: String? = null

    @JsonProperty("server_info")
    private val serverInfo: LinkedHashSet<ServerInfo>? = null
    private val objects: LinkedHashSet<DomainArchitectureMappingObject>? = null
}
